### PR TITLE
Modify ItPodTemplates, ItMiiCustomSslStore class to use v9 Cluster template

### PR DESCRIPTION
**Pull request by anpanigr**
Modify ItPodTemplates, ItMiiCustomSslStore class to use v9 Cluster template

Jenkin Job
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/12508/ ( all passed )
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/12529/

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodTemplates.java [hunk 1]
```diff
@@ -88,88 +89,78 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
...
-      logger.info("Shutting down domain");
-      shutdownDomain(domain1Uid, domain1Namespace);
-    }
+          WDT_BASIC_IMAGE_DOMAINHOME),
+        String.format("testPodTemplateUsingVariablesDomainInImage fails"));
   }
 
   /**
-   * Create domain CRD with added labels and annotations to the serverPod using variables :
+   * Create domain Domain resource with added labels and annotations to the
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodTemplates.java:121]:
```suggestion
   * Create Domain resource with added labels and annotations to the 
```

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodTemplates.java [hunk 2]
```diff
@@ -88,88 +89,78 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
...
           "Image",
           "domain1",
-          WDT_BASIC_IMAGE_DOMAINHOME);
-
-    } finally {
-      logger.info("Shutting down domain");
-      shutdownDomain(domain1Uid, domain1Namespace);
-    }
+          WDT_BASIC_IMAGE_DOMAINHOME),
+        String.format("testPodTemplateUsingVariablesDomainInImage fails"));
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodTemplates.java:117]:
```suggestion
        String.format("creating serverPod template failed"));
```

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodTemplates.java [hunk 3]
```diff
@@ -88,88 +89,78 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
...
+   * Create and verify domain and check that managed server pod labels and 
+   * annotations are added for $(SERVER_NAME), $(DOMAIN_NAME), $(DOMAIN_UID), 
+   * $(DOMAIN_HOME), $(CLUSTER_NAME) and not added for $(LOG_HOME) since 
+   * logHomeEnable option set to false.
    */
-  private void createAndVerifyPodFromTemplate(String imageName, String domainUid, String domainNS,
+  private void createAndVerifyPodFromTemplate(String imageName,
                                               String domainHomeSource,
                                               String domainName,
                                               String domainHome) throws io.kubernetes.client.openapi.ApiException {
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodTemplates.java:133]:
```suggestion
                                              String domainHome) throws ApiException {
```

Comment (sankarpn) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodTemplates.java:133]: import the exception

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodTemplates.java [hunk 4]
```diff
@@ -322,12 +289,23 @@ private static void createDomainCrAndVerify(String adminSecretName,
...
+
+    ClusterSpec clusterSpec = new ClusterSpec()
+            .withClusterName(clusterName)
+            .replicas(replicaCount)
+            .serverPod(new ServerPod()
+                .labels(clusterLabelKeyValues));
+    logger.info(Yaml.dump(clusterSpec));
+    ClusterResource cluster = 
+         createClusterResource(clusterName, domainNamespace, clusterSpec);
+    logger.info("Creating cluster {0} in namespace {1}", clusterName, domainNamespace);
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodTemplates.java:301]:
```suggestion
    logger.info("Creating cluster resource {0} in namespace {1}", clusterName, domainNamespace);
```

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodTemplates.java (hunk 2 above)
Comment (anpanigr) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodTemplates.java:117]: Modified --> Creating Domain resource with serverPod template failed

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodTemplates.java (hunk 3 above)
Comment (anpanigr) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodTemplates.java:133]: done

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodTemplates.java (hunk 4 above)
Comment (anpanigr) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodTemplates.java:301]: done

**Review by sankarpn — APPROVED** [review]
lgtm

**Pull request merged**